bundles/org.openhab.binding.boschshc/src/main/java/org/openhab/binding/boschshc/internal/discovery/ThingDiscoveryService.java

```diff
@@ -59,31 +66,22 @@ public class ThingDiscoveryService extends AbstractDiscoveryService implements T
             new AbstractMap.SimpleEntry<>("TWINGUARD", BoschSHCBindingConstants.THING_TYPE_TWINGUARD),
             new AbstractMap.SimpleEntry<>("PSM", BoschSHCBindingConstants.THING_TYPE_INWALL_SWITCH), // Steckdosen Zwischenstecker
             new AbstractMap.SimpleEntry<>("PLUG_COMPACT", BoschSHCBindingConstants.THING_TYPE_SMART_PLUG_COMPACT),
-            new AbstractMap.SimpleEntry<>("CAMERA_EYES", BoschSHCBindingConstants.THING_TYPE_CAMERA_360),
-            new AbstractMap.SimpleEntry<>("BWTH", BoschSHCBindingConstants.THING_TYPE_WALL_THERMOSTAT), // floorheating room control
+            new AbstractMap.SimpleEntry<>("CAMERA_350", BoschSHCBindingConstants.THING_TYPE_CAMERA_360),
+            new AbstractMap.SimpleEntry<>("CAMERA_EYES", BoschSHCBindingConstants.THING_TYPE_CAMERA_EYES),
+            new AbstractMap.SimpleEntry<>("BWTH", BoschSHCBindingConstants.THING_TYPE_THERMOSTAT), // wall thermostat
```

Comment:
Why are some keys abbreviated and others not? I would align it that we have readable names everywhere if possible.

Reply:
The key is the content of the DeviceModel field inside the [Device SHC REST Schema](https://local.apidocs.bosch-smarthome.com/#/Devices/get_devices)

I not defined the key and we can't change it.

Reply:
Got it, these constants are predefined from Bosch and we cannot change them :+1: